lib/kite/generate.rb

```diff
@@ -34,10 +34,36 @@ def cloud()
       end
     end
 
-    method_option :name, type: :string, desc: "Task name", required: true
-    desc "task", "Generate task IaC from configuration"
-    def task()
-      say "Generating task #{ options[:name] } IaC", :green
+    method_option :force, type: :boolean, desc: "Overwrite existing commands", default: false
+    desc "task COMMAND_NAME [TASK_NAME, ...]", "Generate task IaC from configuration"
```

Comment:
We need to move logic into a class and unit test it,
task generate should remain super simple
We can do a Kite::Generator